packages/common/test/projects/_internal/ProjectUiSchemaEdit.test.ts

```diff
@@ -323,6 +323,60 @@ describe("buildUiSchema: project edit", () => {
             },
           ],
         },
+        {
+          type: "Section",
+          labelKey: `some.scope.sections.embeds.label`,
+          options: {
+            headerTag: "h2",
+            helperText: { labelKey: "some.scope.sections.embeds.helperText" },
+          },
+          elements: [
+            {
+              scope: "/properties/view/properties/embeds",
+              type: "Control",
+              options: {
+                control: "hub-composite-input-embeds",
+                facets: [
```

Comment:
Is there any reason we shouldn't make these the default facets in the actual embed composite field (e.g. when we `buildDefaultFacets`)? I know the field is set up to accept facet and catalog overrides, but I figure we might as well make these the standard default ones directly in the field for now?